backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/emu/com/badlogic/gdx/utils/SerializationException.java

```diff
@@ -19,7 +19,7 @@
 /** Indicates an error during serialization due to misconfiguration or during deserialization due to invalid input data.
  * @author Nathan Sweet */
 public class SerializationException extends RuntimeException {
-	private StringBuffer trace;
+	private StringBuilder trace;

```

Comment:
I can't think of a scenario with the Exception being shared across threads but I'd be cautious with this one just in case

Reply:
Realistically, all of the GWT code is single-thread anyway (the implementations of [StringBuffer](https://gwt.googlesource.com/gwt/+/2.8.2/user/super/com/google/gwt/emul/java/lang/StringBuffer.java) and [StringBuilder](https://gwt.googlesource.com/gwt/+/2.8.2/user/super/com/google/gwt/emul/java/lang/StringBuilder.java) are exactly the same in GWT) so these types of changes in the GWT backend code would only be for consistency purposes.

Reply:
Cool, thanks for the explanation. With this one reviewed, the rest of the changes are safe imo, do you agree @MobiDevelop?

Reply:
Yeah, I don't see anything wrong with changing any of these to StringBuilder.